src/lib/Behat/PageElement/AdminList.php

```diff
@@ -34,24 +37,15 @@ public function __construct(UtilityContext $context, string $listHeader)
         $this->listHeader = $listHeader;
     }
 
-    private function verifyProperList(): void
+    public function verifyVisibility(): void
     {
-        $actualHeader = $this->context->findElement($this->fields['listHeader'], $this->defaultTimeout)->getText();
+        $actualHeader = $this->context->getElementByText($this->listHeader, $this->fields['listHeader']);
 
-        if ($this->listHeader !== $actualHeader) {
+        if ($actualHeader === null) {
             throw new ElementNotFoundException($this->context->getSession(), 'table header', $this->fields['listHeader']);
         }
     }
 
-    public function verifyVisibility(): void
-    {
-        $this->context->waitUntilElementIsVisible($this->fields['plusButton'], $this->defaultTimeout);
```

Comment:
Why is it removed?

Reply:
Because this same structure is also in ContentTypePage, and there's no buttons, only tables with informations, so I leaved only list header checking.